src/reference/00-Getting-Started/11-Custom-Settings.md

```diff
@@ -230,7 +230,7 @@ lazy val library = (project in file("library"))
       println("s: " + s)
       s
     },
-    sampleStringTask := {
+    stopServer := {
```

Comment:
This is not a typo. See in the following `sampleStringTask.value` is assigned to `val old`: If this is called `stopServer` calling it `old` won't make sense. In `build.sbt` you can rewire keys while referring to the old behavior, similar to class inheritance and `super`.

Reply:
😮  oh, will reread it again. thank you.